app/diagnostics.go

```diff
@@ -204,6 +204,7 @@ func trackConfig() {
 		"session_length_mobile_in_days":                    *utils.Cfg.ServiceSettings.SessionLengthMobileInDays,
 		"session_length_sso_in_days":                       *utils.Cfg.ServiceSettings.SessionLengthSSOInDays,
 		"session_cache_in_minutes":                         *utils.Cfg.ServiceSettings.SessionCacheInMinutes,
+		"session_idle_timeout":                             *utils.Cfg.ServiceSettings.SessionIdleTimeout,
```

Comment:
> utils.Cfg.ServiceSettings.SessionIdleTimeout undefined (type model.ServiceSettings has no field or method SessionIdleTimeout)

Did you mean `SessionIdleTimeoutInMinutes`?

Reply:
It's `SessionIdleTimeout` in config.json https://github.com/mattermost/mattermost-server/blob/master/config/default.json#L39

Unless it's a different name elsewhere

Reply:
I think @jwilander goofed. It should be `SessionIdleTimeoutInMinutes` in the config.json too.

https://github.com/mattermost/mattermost-server/pull/7524

Reply:
Oh yeah, I missed updating it in the default config.json. It should be `SessionIdleTimeoutInMinutes`